src/app/admin/_components/Organization/Create.js

```diff
@@ -0,0 +1,84 @@
+import {
+  BooleanInput,
+  Create,
+  FormDataConsumer,
+  NullableBooleanInput,
+  NumberInput,
+  SelectArrayInput,
+  SelectInput,
+  SimpleForm,
+  TextInput,
+  required,
+} from 'react-admin';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { OrganizationSchema } from '@admin/_lib/validationSchemas/createOrganizationSchema';
+import { transformOrganizationData } from '@admin/_utils/transformOrganizationData';
+import { ORGANIZATION_TYPES } from '@admin/_lib/consts';
+import { AddressInput } from './CreateAdresses';
+import { SelectTherapies } from './SelectTherapies';
+
+const fieldGroupClass = 'flex flex-col md:flex-row md:gap-6';
+
+export function OrganizationCreate() {
+  return (
+    <Create transform={transformOrganizationData}>
+      <SimpleForm resolver={zodResolver(OrganizationSchema)}>
+        <FormDataConsumer>
+          {({ formData }) => (
+            <>
+              <p className="font-bold">Основна інформація</p>
+              <SelectArrayInput
+                source="type"
+                choices={ORGANIZATION_TYPES.map(type => ({ id: type, name: type }))}
```

Comment:
Please take the organization types from the database instead of storing it as a const. You can use Specialist create form as a reference.